Source/Bot/BotStatistics.swift

```diff
@@ -58,6 +58,9 @@ extension BotStatistics {
 /// Statistics for the go-ssb log
 struct RepoStatistics: Equatable {
 
+	/// Identity associated with the repo
+    let identity: Identity?
```

Comment:
I couldn't find where this is set as the code is still creating RepoStatistics objects without setting the identity

Reply:
Thanks for the head's up. I must have stashed it or something. `identity` is now set here: 

https://github.com/planetary-social/planetary-ios/blob/bf6b617ae23769c5f8476f0baa99ea98418652ef/Source/GoBot/GoBot.swift#L1603